doc/Makefile.am

```diff
@@ -148,6 +148,75 @@ CLEAN_LOCAL_TARGETS += clean-gtkdoc-header-local
 clean-gtkdoc-header-local:
 	-rm -rf xml/ Doxyfile-gi Doxyfile-gi.stamp geany-gtkdoc.h geany-sciwrappers-gtkdoc.h
 
+if HAVE_INTROSPECTION
+
+INTROSPECTION_GIRS  = Geany-1.0-pre.gir GeanyScintilla-1.0-pre.gir
```

Comment:
This is so that `Geany.sci_start_undo_action(sci)` works at the same time (on the same object instance) as GeanyScintilla's `sci.start_undo_action()`.

For this to work, `Geany.sci_start_undo_action()` must take a GeanyScintilla.ScintillaObject parameter (same type as `sci.start_undo_action()`)